status_service/status_checks.py

```
@@ -124,9 +124,40 @@ def check_ssl_status():
 
-def check_nginx_status():
-    host = os.getenv("STATUS_NGINX_HOST", "nginx")
-    port = int(os.getenv("STATUS_NGINX_PORT", "80"))
+def _resolve_proxy_target():
+    host = "traefik"
+    host_env_var = None
+    for env_var in ("STATUS_PROXY_HOST", "STATUS_TRAEFIK_HOST", "STATUS_NGINX_HOST"):
+        value = os.getenv(env_var)
+        if value:
+            host = value
+            host_env_var = env_var
```

Ruff (0.14.14)

[error] Local variable `host_env_var` is assigned to but never used

Remove assignment to unused variable `host_env_var`

(F841)